libr/include/r_asm.h

```diff
@@ -267,6 +267,7 @@ extern RAsmPlugin r_asm_plugin_xap;
 extern RAsmPlugin r_asm_plugin_xcore_cs;
 extern RAsmPlugin r_asm_plugin_xtensa;
 extern RAsmPlugin r_asm_plugin_z80;
+extern RAsmPlugin r_asm_plugin_llvm;
```

Comment:
not sure if we want to name it llvm or bitcode or llil, comments here?

Reply:
llvm is fine, llil is misleading, bitcode is too vague